src/mapgen/mg_ore.h

```diff
@@ -173,7 +173,7 @@ class OreManager : public ObjDefManager {
 		case ORE_STRATUM:
 			return new OreStratum;
 		default:
-			return nullptr;
+			return new OreScatter;
```

Comment:
This is logically wrong. Different ideas:
* Remove the default case and add `return nullptr;` outside the `switch` statement. This will result in compiler warnings if a case was not handled (e.g. forgotten).
* Same as above but throw a C++ exception. The Lua side will catch and display that properly.

Reply:
Yeah, removing the default case is the right way here.
I also replaced it with a Lua exception, but the create function can stay where it is, I think.